tecken/api/views.py

```diff
@@ -598,92 +597,31 @@ def stats(request):
             return sub_qs.aggregate(count=Count("id"), total_size=Sum("size"))
 
-        def count(qs, start, end):
-            sub_qs = qs.filter(created_at__gte=start, created_at__lt=end)
-            return sub_qs.aggregate(count=Count("id"))
-
         numbers["uploads"] = {
             "all_uploads": all_uploads,
             "today": count_and_size(upload_qs, start_today, today),
             "yesterday": count_and_size(upload_qs, start_yesterday, start_today),
             "last_30_days": count_and_size(upload_qs, last_30_days, today),
         }
-        numbers["files"] = {
-            "today": count(files_qs, start_today, today),
-            "yesterday": count(files_qs, start_yesterday, start_today),
-            "last_30_days": count(files_qs, last_30_days, today),
-        }
 
-    with metrics.timer("api_stats", tags=["section:all_missing_downloads"]):
-        # When doing aggregates on rows that don't exist you can get a None instead
-        # of 0. Only really happens in cases where you have extremely little in the
-        # database.
-        def nones_to_zero(obj):
-            for key, value in obj.items():
-                if isinstance(value, dict):
-                    nones_to_zero(value)
-                elif value is None:
-                    obj[key] = 0
-
-        nones_to_zero(numbers)
-
-        missing_qs = MissingSymbol.objects.all()
-
-        def count_missing(start, end, use_cache=True):
-            count = None
-            if use_cache:
-                fmt = "%Y%m%d"
-                cache_key = f"count_missing:{start.strftime(fmt)}:{end.strftime(fmt)}"
-                count = cache.get(cache_key)
-            if count is None:
-                qs = missing_qs.filter(modified_at__gte=start, modified_at__lt=end)
-                count = qs.count()
-                if use_cache:
-                    cache.set(cache_key, count, 60 * 60 * 24)
-            return {"count": count}
-
-        numbers["downloads"] = {
-            "missing": {
-                "today": count_missing(start_today, today, use_cache=False),
-                "yesterday": count_missing(start_yesterday, start_today),
-                "last_30_days": count_missing(last_30_days, start_today),
-            },
-        }
-        # A clever trick! Instead of counting the last_30_days to include now,
-        # we count the last 29 days instead up until the start of today.
-        # Then, to make it the last 30 days we *add* the "today" count.
-        numbers["downloads"]["missing"]["last_30_days"]["count"] += numbers[
-            "downloads"
-        ]["missing"]["today"]["count"]
-
-    with metrics.timer("api_stats", tags=["section:your_tokens"]):
-        # Gather some numbers about tokens
-        tokens_qs = Token.objects.filter(user=request.user)
-        numbers["tokens"] = {
-            "total": tokens_qs.count(),
-            "expired": tokens_qs.filter(expires_at__lt=today).count(),
-        }
```

Comment:
This isn't in the frontend anywhere. I didn't look at the logs to figure out when we took it out.